libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -1,43 +1,48 @@
+import classNames from 'classnames';
 import { create } from 'zustand';
 import {
-  Button,
   Dialog,
   FormGroup,
   Input,
+  Intent,
+  Lozenge,
+  TradingButton,
   VegaIcon,
   VegaIconNames,
 } from '@vegaprotocol/ui-toolkit';
+import type { ReactNode } from 'react';
 import { useCallback, useState } from 'react';
 import type { WalletClientError } from '@vegaprotocol/wallet-client';
 import { t } from '@vegaprotocol/i18n';
 import type { VegaConnector } from '../connectors';
-import { InjectedConnector } from '../connectors';
-import { ViewConnector } from '../connectors';
-import { JsonRpcConnector, RestConnector } from '../connectors';
-import { RestConnectorForm } from './rest-connector-form';
+import { InjectedConnector, JsonRpcConnector } from '../connectors';
 import { JsonRpcConnectorForm } from './json-rpc-connector-form';
-import { Networks, useEnvironment } from '@vegaprotocol/environment';
+import { useEnvironment } from '@vegaprotocol/environment';
 import {
   ConnectDialogContent,
   ConnectDialogFooter,
   ConnectDialogTitle,
+  BrowserIcon,
+  CHROME_EXTENSION_URL,
+  MOZILLA_EXTENSION_URL,
 } from './connect-dialog-elements';
 import type { Status as JsonRpcStatus } from '../use-json-rpc-connect';
-import type { Status as InjectedStatus } from '../use-injected-connector';
 import { useJsonRpcConnect } from '../use-json-rpc-connect';
-import { ViewConnectorForm } from './view-connector-form';
+import type { Status as InjectedStatus } from '../use-injected-connector';
+import { useInjectedConnector } from '../use-injected-connector';
 import { useChainIdQuery } from './__generated__/ChainId';
 import { useVegaWallet } from '../use-vega-wallet';
-import { useInjectedConnector } from '../use-injected-connector';
 import { InjectedConnectorForm } from './injected-connector-form';
+import { isBrowserWalletInstalled } from '../utils';
+import { useIsDesktopWalletRunning } from '../use-is-desktop-wallet-running';
```

Comment:
Everywhere we refer to isDesktopWalletRunning can we rename this to..

`isWalletServiceRunning` and `useIsWalletServiceRunning`. As its the same for clie and dw wallets